wagtail/admin/static_src/wagtailadmin/scss/components/_listing.scss

```diff
@@ -483,9 +490,7 @@ table.listing {
     }
 
     .table-headers {
-        .ord {
-            padding-right: 0;
-        }
+        height: 35px;
```

Comment:
The height of the header row was dependent on the reordering icon. This makes it the same height whether reordering is enabled or not.